sofa-ark-parent/support/ark-springboot-starter/src/test/java/com/alipay/sofa/ark/runner/RunnerHelper.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.ark.runner;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author qilong.zql
+ * @since 0.1.0
+ */
+public class RunnerHelper {
+
+    public static AtomicBoolean isHelped = new AtomicBoolean(false);
+
+    public static void prepare() {
+        try {
+            if (isHelped.compareAndSet(false, true)) {
+                String shPath = RunnerHelper.class.getClassLoader()
+                    .getResource("prepareForTest.sh").getPath();
```

Comment:
Why execute a script here?

Reply:
when execute `mvn test` or run test case in ide， `target/classes` of `sofa-ark-all` ，not the jar file, is contained in classpath； the script will package a sofa-ark-all.jar ， and will be put into classpath...  Is'a ugly solution, but only used when run  test case of ark. so i plan to improve it in next version. Just provide a `DirectoryArchiver` used to parse ark container represented in format of directory.

Reply:
OK~